app/src/main/java/com/jyoti/androidtutorial/home/data/data_source/HomeRemoteDataSource.kt

```diff
@@ -0,0 +1,13 @@
+package com.jyoti.androidtutorial.home.data.data_source
+
+import com.jyoti.androidtutorial.home.data.model.HomeResponseModel
+import retrofit2.Call
+import retrofit2.http.GET
+
+
+interface HomeRemoteDataSource {
+
+    @GET("/home")
+    fun fetchHomeDataRemote(): Call<HomeResponseModel?>?
```

Comment:
can the Call<> be null ???

Reply:
No, it can not be null. Thank you for pointing this.